packages/webapp/src/pages/Connection/List.tsx

```diff
@@ -175,66 +179,112 @@ export default function ConnectionList() {
-                                            id !== connections.at(-1)?.id ? 'border-b border-border-gray' : ''
-                                        } min-h-[4em] px-2 justify-between items-center hover:bg-hover-gray cursor-pointer`}
-                                        onClick={() => {
-                                            navigate(`/${env}/connections/${encodeURIComponent(providerConfigKey)}/${encodeURIComponent(connectionId)}`);
-                                        }}
-                                    >
-                                        <div className="flex items-center w-2/3 gap-2 py-2 truncate">
-                                            <span className="break-words break-all truncate">{connectionId}</span>
-                                            {active_logs && <ErrorCircle />}
-                                            <CopyButton dark text={connectionId} />
-                                        </div>
-                                        <div className="flex items-center w-1/3 gap-3">
-                                            <div className="w-7">
-                                                <IntegrationLogo provider={provider} height={7} width={7} />
+                            <div className="rounded-md overflow-hidden">
+                                {filteredConnections.map(
+                                    ({
+                                        id,
+                                        connection_id: connectionId,
+                                        provider,
+                                        provider_config_key: providerConfigKey,
+                                        created: creationDate,
+                                        active_logs,
+                                        connection_config
+                                    }) => (
+                                        <div
+                                            key={`tr-${id}`}
+                                            className={`flex gap-4 ${
+                                                id !== connections.at(-1)?.id ? 'border-b border-border-gray' : ''
+                                            } min-h-[4em] px-2 justify-between items-center hover:bg-hover-gray cursor-pointer`}
+                                            onClick={() => {
+                                                navigate(`/${env}/connections/${encodeURIComponent(providerConfigKey)}/${encodeURIComponent(connectionId)}`);
+                                            }}
+                                        >
+                                            <div className="flex items-center w-1/4 gap-2 py-2 truncate">
+                                                <span className="break-words break-all truncate">
+                                                    {connection_config?.display_name ? (connection_config.display_name as string | undefined) : '-'}
```

Comment:
we probably need to fallback to connectionId because right now it's not usable for legacy connection 
<img width="657" alt="Screenshot 2024-07-08 at 11 46 55" src="https://github.com/NangoHQ/nango/assets/1637651/2b7c70cf-c7d3-405d-8ddb-43943085a6ae">

Reply:
Feel like if the user isn't using these fields then we shouldn't show these rows at all. To make it more discoverable it would be nice to add in optional fields in the Connection/Create screen